docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -122,6 +122,43 @@ Elliptic Curve Signature Algorithms
         :returns: A new instance of a :class:`EllipticCurvePublicKey`
             provider.
 
+Elliptic Curve Key Exchange algorithm
+-------------------------------------
+
+.. class:: ECDH(algorithm)
```

Comment:
As said, you can pass a NullKDF() to avoid key derivation. I think this is a better pattern, however ECDH() is also hazmat so if @alex and @reaperhulk agree we can allow to pass None and have it automatically ignored withing the code. They suggested to make it mandatory so I'd defer to them.